h2o-persist-gcs/build.gradle

```diff
@@ -4,7 +4,7 @@ description = "H2O Persist GCS"
 
 dependencies {
     compile project(":h2o-core")
-    compile 'com.google.cloud:google-cloud-storage:1.19.0'
+    compile 'com.google.cloud:google-cloud-storage:1.111.1'
```

Comment:
Version `1.111.1` of GCS carries jackson-core and jackson-databind version `2.11.1` with it.